app/javascript/react/screens/App/Mappings/components/InfrastructureMappingsList/InfrastructureMappingsList.js

```diff
@@ -117,9 +117,12 @@ class InfrastructureMappingsList extends React.Component {
       <React.Fragment>
         <ListViewToolbar
           filterTypes={INFRA_MAPPINGS_FILTER_TYPES}
+          defaultFilterTypeIndex={0}
```

Comment:
I think it would be nice if the index always defaults to zero, but can be overridden if desired by passing a prop

Reply:
Fixed in c3a25c620f006e910da3bd387f29a7b7ce8063d4 using `defaultProps`.